jest-es-config.js

```diff
@@ -0,0 +1,30 @@
+const documentsMapping = require('./documents-mapping');
+
+module.exports = function generateMapping(
+  mappingSchema,
+  indexName,
+  aliasName,
+  elasticSearchVersion
+) {
+  return {
+    esVersion: elasticSearchVersion || '6.8.2',
+    clusterName: 'docs',
+    nodeName: 'docs',
+    port: 9200,
+    indexes: [
+      {
+        name: indexName || 'documents',
```

Comment:
```suggestion
        name: indexName,
```

Defaulting here will be very unexpected for library users